src/cockpit/router.py

```diff
@@ -26,12 +27,54 @@
 logger = logging.getLogger(__name__)
 
 
+class ExecutionQueue(collections.deque):
+    """Temporarily delay calls to a given set of class methods.
+
+    Functions by replacing the named function at the instance __dict__
+    level, effectively providing an override for exactly one instance
+    of `method`'s object.
+    Queues the invocations.  Run them later with .run(). Reverse this call
+    by deleting the named methods from the instance.
```

Comment:
That last sentence is not true any more.

Reply:
Fixed.